src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -209,6 +209,28 @@ message PollOperationRequest {
   build.bazel.remote.execution.v2.ExecutionStage.Value stage = 3;
 }
 
+// Publish GRPC metrics to promethesus for each server endpoint. This is used
+// for SLAs at API boundaries. You may want something similar at the proxy level
+// to see frontend/backend performance.
+message GrpcPrometheusMetrics {
```

Comment:
Does buildfarm server serve anything other than GRPC? I think there is a proxy, but is prefixing with GRPC a bit redundant if its the only protocol buidlfarm serves.

Reply:
There is a similar grpc server built in `buildfarm/proxy/http/HttpProxy.java`.  It could get the same configuration in the future if there is a need.  I haven't used that proxy before.  Our load balancers (haproxy) forward all the traffic. Yes, all the Prometheus metrics are prefixed with `grpc_*`.  That's from the intercept library deciding to do that.